src/Server/SSH/SSHPtyHandler.cpp

```diff
@@ -485,14 +505,16 @@ void SSHPtyHandler::run()
 
 
     sdata.channel_callback->channel.sendEof();
+    sdata.channel_callback->channel.sendExitStatus(0);
```

Comment:
Should we sent real status? I.e. non-zero for exceptions?

Reply:
Yes, looks like fixed.